### ISSUE TITLE
`--app-data` flag is not being honored

### ISSUE THREAD

**Issue by luevano** [recyclarr/recyclarr#284]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The `--app-data` flag is not being honored and it is creating the default directories/data on the default paths. I have it set to `/var/lib/recyclarr/`.

EDIT: current dirty fix is to make a symbolic link (`ln -s /var/lib/recyclarr/ /var/lib/recyclarr/.config/recyclarr`), until this is fixed.

### Expected Behavior

It should use the specified `--app-data` dir. Before version `7.0.0` it was working as intended.

### Steps To Reproduce

1. Have the user `recyclarr` with its home directory at `/var/lib/recyclarr/`.
2. Run recyclarr as this user and use  `--app-data /var/lib/recyclarr/`.
3. It will create the `.config/recyclarr` directory inside of `/var/lib/recyclarr/` like this: `/var/lib/recyclarr/.config/recyclarr` and all default dirs/data here.
4. It wont find the already defined `secrets.yml` and configs.

### Environment & Versions

```bash
- Operating System: Arch Linux on 6.9.7-arch1-1
- Architecture: x86_64
- Recyclarr Version: v7.0.0 (Branch.master.Sha.119808bf1408948718c002eeaae8403dde371b67)
- Recyclarr Build: linux-x64
- Sonarr Version: -
- Radarr Version: -
```

### Recyclarr Log File(s)

Working (6.0.2):
[working_recyclarr_6.0.2.txt](https://github.com/user-attachments/files/16123317/working_recyclarr_6.0.2.txt)

Failing (current 7.0.0):
[failing_recyclarr_7.0.0.txt](https://github.com/user-attachments/files/16123318/failing_recyclarr_7.0.0.txt)

### Configuration YAML

```YAML
-
```

**Comment by rcdailey** [comment]
Thank you for the report! This will be available in v7.1.0, which is in the pipe right now.

**Comment by luevano** [comment]
> Thank you for the report! This will be available in v7.1.0, which is in the pipe right now.

Sounds good, thanks for the quick fix!
